wordpress-mobile/WordPress-iOS
# Update to AppCenter SDK

This PR removes the old HockeyApp SDK in favour of the new AppCenter one. 

### To test:
1. Make sure CI is green.
2. Change `VERSION_SHORT` and `VERSION_LONG` strings in `Version.internal.xcconfig` to an older value.
3. Build the app and upload to AppCenter.
4. Download the version you just built on your device from AppCenter and release it to a distribution group you are in (`Collaborators` should work). Make sure you don't release to yourself alone: you need to release to a group in order to be able to register for updates.
5. Start the app and register for in-app updates following the instructions.
6. Close and re-open the app and make sure the update prompt is shown.
7. Start the in-app update and verify it works without errors.

Note: Version 13.6.0.201911151 is available on AppCenter and has been built out of this branch for testing. If you want to use it, you just have to enable it on AppCenter (click on the ellipses and then `Enable Release`) to make it available for download.

PR submission checklist:
